test/message/core_line_numbers.chakracore.out

```diff
@@ -1,5 +1,5 @@
 RangeError: Invalid input
-   at error (punycode.js:67:*)
+   at error (punycode.js:42:*)
```

Comment:
This is intended to make sure that line number in `punycode` module doesn't change. See `core_line_numbers.js` for details.